src/api/app/services/notification_service/web_channel.rb

```diff
@@ -0,0 +1,18 @@
+module NotificationService
+  class WebChannel
+    def initialize(subscription, event)
+      @subscription = subscription
+      @event = event
+    end
+
+    def call
+      params = @subscription.parameters_for_notification.merge!(@event.parameters_for_notification)
+      notification = Notification.find_by(notifiable_type: params[:notifiable_type], notifiable_id: params[:notifiable_id], web: true)
+      OutdatedNotifications::Destroyer.new(@subscription.subscriber, notification).call if notification
+
+      notification = Notification.create(params)
```

Comment:
if will explode if notification is nil.. `Notification.create!` or just call `.projects` if  `notification` exists

Reply:
We are making heavy changes to this section, but I will take it into account, thanks!